acvm-repo/acvm/src/compiler/transformers/mod.rs

```diff
@@ -42,7 +43,7 @@ pub(super) fn transform_internal(
     expression_width: ExpressionWidth,
     acir_opcode_positions: Vec<usize>,
 ) -> (Circuit, Vec<usize>) {
-    log::trace!("Start circuit transformation");
+    trace!("Start circuit transformation");
```

Comment:
Looks like we'd want to replace the these with a span https://docs.rs/tracing/latest/tracing/#spans

Reply:
we could but don't have to...

Reply:
I think it makes sense to add it, another strategy is to just annotate the method with `#[instrument]` because the span being created is for the whole function

Reply:
https://docs.rs/tracing-attributes/latest/tracing_attributes/attr.instrument.html

Reply:
in follow up PR?

Reply:
This is a 5 min job isn't it? Might as well roll it into this PR.

Reply:
I don't think it is...

Reply:
@kobyhallx what seems to be difficult about this task? From the documentation this is a simple copy and paste job

Reply:
I've made these changes to the PR this morning.